pai/job/_training_job.py

```diff
@@ -708,10 +708,7 @@ def _training_job_base_output(self, job_name):
             StoragePathCategory.TrainingJob,
             f"{to_plain_text(job_name)}_{random_str(6)}",
         )
-        base_output_path = (
-            f"oss://{bucket_name}.{session.oss_endpoint}/"
-            f"{storage_path}"
-        )
+        base_output_path = f"oss://{bucket_name}/{storage_path}"
```

Comment:
Remove the OSS endpoint from the URI, as the backend service supports the OSS URI pattern that does not include the endpoint.